script.js

```diff
@@ -55,23 +39,34 @@ class World {
     gameField.appendChild(object.element);
     this.playFieldObjects.push(object);
   }
+
+  anyCollisionsWith(entity) {
+    const offset = {
+      x: 9,
+      y: 9
+    }
+
+    return this.playFieldObjects.some(playFieldObject => {
+      return this.willCollide(playFieldObject, entity, offset);
+    });
+  }
+
+  willCollide(rect1, rect2, offset) {
+    return rect1.x < rect2.x + rect2.width &&
```

Comment:
What do you think of adding an offset of like -50 to rect2.width to make up for the large sprite area? Or make an adjustable property like offset to the player object?
`return rect1.x < rect2.x + (rect2.width - 50) &&`

Reply:
That's a good idea. we could just leverage the overflow properties of the images though to scale them past the actual div

Reply:
nevermind, didn't realize you used background for that, i'll slap that in as a property on the player object

Reply:
see: https://github.com/I3uckwheat/attack-of-the-clowns/pull/10#discussion_r277139479